### PR TITLE
tiles view and view refactoring

### PR DESCRIPTION
**Pull request by arturAR**
View refactoring. 
Tiles view.

### CODE REVIEW

File: src/main/java/com/java/academy/controller/MainController.java [hunk 1]
```diff
@@ -31,21 +31,20 @@ public String welcome(Model model) {
...
 	public String books(Model model) {
-		model.addAttribute("books", bookService.getAllBooks());
 
 		return "books";
 	}
 
-	@RequestMapping("/books2")
+	@RequestMapping("/booksTiles")
 	public String books2(Model model) {
-		model.addAttribute("books", bookService.getAllBooks());
```
Comment (mprtcz) [inline src/main/java/com/java/academy/controller/MainController.java:43]: Why empty line?

File: src/main/java/com/java/academy/controller/MainController.java [hunk 2]
```diff
@@ -31,21 +31,20 @@ public String welcome(Model model) {
...
 
-		return "books2";
+		return "booksTiles";
 	}
 
 	@RequestMapping(value = "/rest/startBooks", method = RequestMethod.GET)
 	public @ResponseBody List<Book> readStartBooks() {
-		return bookService.getBooksByFilter("category", "book");
+		List<Book> books = bookService.getBooksByFilter("category", "book");
```
Comment (mprtcz) [inline src/main/java/com/java/academy/controller/MainController.java]: Enum would be better

File: src/main/java/com/java/academy/controller/MainController.java [hunk 3]
```diff
@@ -31,21 +31,20 @@ public String welcome(Model model) {
...
+		return books;
 	}
 
 	@RequestMapping(value = "/rest/books/{filter}/{data}", method = RequestMethod.GET)
```
Comment (mprtcz) [inline src/main/java/com/java/academy/controller/MainController.java]: GET is a default one, so @RequestMapping("/rest/books/{filter}/{data}")

File: src/main/webapp/WEB-INF/templates/fragments/footer.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <meta charset="UTF-8">
+    <title>footer</title>
+</head>
+<body>
+    <div class="navbar navbar-default navbar-fixed-bottom" th:fragment="footer">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-6">
+                    <p class="navbar-text">
+                        Copyright © Paweł & Mateusz & Artur & Wiktoria. All rights reserved.
```
Comment (mprtcz) [inline src/main/webapp/WEB-INF/templates/fragments/footer.html:13]: 4 people team. That's cheating :hankey:

**Review by mprtcz — APPROVED** [review]

File: src/main/webapp/WEB-INF/templates/fragments/footer.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+                    </p>
+                </div>
+                <div class="col-lg-6">
+                    <div class="navbar-text navbar-right">
+                        <a href="https://www.facebook.com/MumJustOneMoreChapter/"><i id="social-fb" class="fa fa-facebook-square fa-2x social"></i></a>
+                    </div>
```
Comment (patkkka) [inline src/main/webapp/WEB-INF/templates/fragments/footer.html:19]: 👍

**Review by patkkka — APPROVED** [review]

**Pull request merged**